cinema/views.py

```diff
@@ -0,0 +1,41 @@
+from django.shortcuts import get_object_or_404
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+
+
+from .models import Movie
+from .serializers import MovieSerializer
+

```

Comment:
It's a good practice to include a blank line between import statements and the rest of the code to improve readability.